versioned_docs/version-8.7/components/concepts/process-instance-migration.md

```diff
@@ -362,6 +362,14 @@ In the following cases, the process instance can't apply the migration plan and
 - Multi-instance body limitations:
   - Each child instance of a multi-instance body should be migrated separately because they belong to another process instance.
   - It is not possible to migrate a parallel multi-instance body to a sequential multi-instance body and vice versa.
+- Joining parallel gateway limitations:
+  - All taken sequence flow from the source needs to be mapped in the target definiton
+  - All taken sequence flow needs to be mapped to the same type of gateway
```

Comment:
Singular / plural is mixed up here.

Do you want to say:
* All taken sequence flow**s** have to be mapped to the target definition?

To be honest: these two points are hard to understand. I fail to understand what you are trying to say :-)